src/components/InfoPanel/InfoPanel.js

```diff
@@ -36,6 +36,7 @@ const InfoPanel = () => {
   const { 0: modelName } = useParams();
   const [showExpandedTopology, setShowExpandedTopology] = useState(false);
   const modelStatusData = useModelStatus();
+  const applicationsCount = Object.entries(modelStatusData.applications).length;
```

Comment:
it's possible that `modelStatusData.applications` is not defined. We should probably have the reducer set sane defaults for the top level keys like this.
```suggestion
  const applicationsCount = Object.entries(modelStatusData.applications || {}).length;
```